src/rcore.c

```diff
@@ -4926,7 +4944,7 @@ void PollInputEvents(void)
 
     // Register previous mouse states
     CORE.Input.Mouse.previousWheelMove = CORE.Input.Mouse.currentWheelMove;
-    CORE.Input.Mouse.currentWheelMove = 0.0f;
+    CORE.Input.Mouse.currentWheelMove = (Vector2){0.0f, 0.0f};
```

Comment:
Please, could you add space after the parentesis, it's a raylib convention I use: `(Vector2){ 0.0f, 0.0f }`